run-app/README.md

```diff
@@ -4,29 +4,65 @@
 
 ## Maven profiles
 
-1. **start-servers:** Starts all liberty servers. 
+1. **start-servers:** Starts all liberty servers
 1. **start-databases:** Starts all mongoDBs
-1. **stop-servers:** Stops all liberty servers. 
+1. **stop-servers:** Stops all liberty servers
 1. **stop-databases:** Stops all mongoDBs
 1. **demo:** Preloads Acme Gifts with pertinent data. Use it only after all databases and servers have been started.
    
-    Example Usage:
-    
-     - From the sample-acmegifts/run-app project:
-        - mvn package -P start-databases
-        - mvn package -P start-servers
-        - mvn package -P demo
-        - mvn package -P start-databases,start-servers,demo
-        - mvn package -P stop-servers,stop-databases
-        
-     - From the sample-acmegifts root project:
-        - mvn package -P start-databases -pl run-app
-        - mvn package -P start-servers -pl run-app
-        - mvn package -P demo -pl run-app
-        - mvn package -P start-databases,start-servers,demo -pl run-app
-        - mvn package -P stop-servers,stop-databases -pl run-app
-
+### Example Usage:
+
+From the sample-acmegifts/run-app project:
+```
+mvn package -P start-databases  
+mvn package -P start-servers  
+mvn package -P demo  
+mvn package -P start-databases,start-servers,demo  
+mvn package -P stop-servers,stop-databases  
+```
+
+From the sample-acmegifts root project:
+```
+mvn package -P start-databases -pl run-app  
+mvn package -P start-servers -pl run-app  
+mvn package -P demo -pl run-app  
+mvn package -P start-databases,start-servers,demo -pl run-app  
+mvn package -P stop-servers,stop-databases -pl run-app  
+```
+
+## Gradle tasks
+
+1. **startServers:** Start all liberty servers
```

Comment:
To follow the wording of all the other ones, change to `Starts all liberty servers`